app/routes/search/publication-flows.js

```diff
@@ -109,6 +118,10 @@ export default class PublicationFlowSearchRoute extends Route {
       filter[':terms:statusId'] = params.publicationStatusIds;
     }
 
+    if (params.urgentOnly) {
+      filter[':terms:urgencyLevelId'] = this.urgencyLevelSpeed.id;
```

Comment:
I think the filter here needs to be `:terms:urgencyLevelId.keyword`. Locally, I can't get the search to only return results marked as urgent. After appending `.keyword` to the filter, it does seem to work as expected.

I assume it's related to how Elasticsearch searches string fields. See this conversation in chat for reference: https://chat.semte.ch/channel/mu-semtech?msg=KEHQtfdWyLWEHj7f5